api/tutorials/token_config.py

```diff
@@ -0,0 +1,26 @@
+from subprocess import run
+# untrack the change to this file (prevent pushing token to github)
+run(['git', 'update-index', '--assume-unchanged', __file__])
```

Comment:
This line will prevent the changes to this file to be tracked by git upon the first time a user does `from token_config import get_oep_token()`.

As experienced user will prefer the environment variable option, I assumed that the chances a user commit and push their change after copy-pasting their token in the file to be pretty slim.